documentation/environment-operations-center-guide/secure-data-connectors/deploy-sdc-client.md

```diff
@@ -80,18 +80,177 @@ To deploy the secure data connector client on a Linux system, first unzip the *s
 - *appsettings.Production.json*
 - *RadiantLogic.OnPremisesAgentClient.Agent.exe*
 
-Open the *appsettings.Production.json* file and locate the `"AgentToken"` field. Enter the token copied from the *Data Connector Registration* dialog in Env Ops Center into the `"AgentToken"` field.
+Open the *appsettings.Production.json* file using an editor and locate the `"AgentToken"` field. Enter the token copied from the *Data Connector Registration* dialog in Env Ops Center into the `"AgentToken"` field.
 
-![image description](images/appsettings-token.png)
+![image description](images/linux-vi-appsettingsjson-file.png)
 
 If the client to be run on a network where proxy is setup, see the *Proxy Configuration for Linux* section in [run the secure data connector client under proxy network setup](deploy-sdc-client-in-proxy.md) guide.
 
+To deploy the sdc client on Linux, the server need to have the .NET core runtime. Depending on the Linux distribution, the installation steps can be found on the [official Microsoft website](https://learn.microsoft.com/en-us/dotnet/core/install/linux). To validate .NET is installed successfully, open a command line and run the dotnet command:
+
+`dotnet`
+
+The output result from the command line would confirm .NET is installed correctly.
+
+![image description](images/.net-installed.png)
+
 Open the command line and navigate to the directory that contains the unzipped *sdc-linux.zip* files. From the directory, run the following command to launch the secure data connector client:
 
-`dotnet run RadiantLogic.OnPremisesAgentClient.Agent.dll`
+`dotnet RadiantLogic.OnPremisesAgentClient.Agent.dll`
+
+A notification will display in the command line that confirms a connection has been established between the agent and server.
+
+![image description](images/linux-dotnet-run-client.png)
```

Comment:
Can you make sure, inlets-pro process is getting fired, we will have to give execute permissions for RadiantLogic.OnPremisesAgentClient.Agent and inlets-pro applications for them to run, similar to what we have in "running as daemon".